replicatedhq/ship
# Kustomize usability

What I Did
------------
Made the kustomize page a lot more usable

How I Did it
------------
Cleaned up a lot of state management, added a diff, overall style and functionality polish

How to verify it
------------
Use it and you'll see things are working far better

Description for the Changelog
------------
Kustomize overlays page cleanup and usability enhancements.


Picture of a Boat (not required but encouraged)
------------
